pump/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for pump 1.0
+// Project: https://github.com/mafintosh/pump
+// Definitions by: Tomek Łaziuk <https://github.com/tlaziuk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function pump(streams: pump.Stream[], callback?: pump.Callback): pump.Stream[];
+
+// callback have to be passed as last argument
+declare function pump(...streams: Array<pump.Stream | pump.Callback>): pump.Stream[];
```

Comment:
actually maybe it is possible, but I don't know how to achieve that, that's very rare to put rest parameters at first

I've been looking for something like typed array with index -1 type, but such thing doesn't exists